doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -558,6 +560,36 @@ semanage port -a -t postgresql_port_t -p tcp 5433
 systemctl enable postgresql-serverconf
 ```
 
+##### 4.2.1.3 on RHEL PostgreSQL 13
+
+Create a new `systemctl` service unit for the new database. As root, make a copy for the new service
+
+```bash
+cp /lib/systemd/system/postgresql-13.service /etc/systemd/system/postgresql-serverconf.service 
+```
+
+Edit `/etc/systemd/system/postgresql-serverconf.service` and override the following properties:
+
+```properties
+[Service]
+...
+Environment=PGPORT=5433
+Environment=PGDATA=/var/lib/pgsql/13/serverconf
+```
+
+Create the database and configure SELinux:
+
+```bash
+# Init db
+sudo su postgres
+cd /tmp
+/usr/pgsql-13/bin/initdb --auth-local=peer --auth-host=md5 --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/pgsql/13/serverconf/
```

Comment:
md5 is not considered safe these days, maybe it can be replaced with sha256?
```suggestion
/usr/pgsql-13/bin/initdb --auth-local=peer --auth-host=scram-sha-256 --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/pgsql/13/serverconf/
```